From 658273fd043f1ad562ef24efd9eb2842862e6204 Mon Sep 17 00:00:00 2001
From: chewitt <github@chrishewitt.net>
Date: Sat, 10 Aug 2019 19:19:00 +0000
Subject: [PATCH 149/182] WIP: arm64: dts: meson-g12b-khadas-vim3: disable LED
 heartbeat

The constantly flashing white LED heartbeat is annoying for an HTPC device
so remove the heartbeat line.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 arch/arm64/boot/dts/amlogic/meson-g12b-khadas-vim3.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/amlogic/meson-g12b-khadas-vim3.dtsi b/arch/arm64/boot/dts/amlogic/meson-g12b-khadas-vim3.dtsi
index 5a84185..0cac7bb 100644
--- a/arch/arm64/boot/dts/amlogic/meson-g12b-khadas-vim3.dtsi
+++ b/arch/arm64/boot/dts/amlogic/meson-g12b-khadas-vim3.dtsi
@@ -96,7 +96,6 @@
 		white {
 			label = "vim3:white";
 			gpios = <&gpio_ao GPIOAO_4 GPIO_ACTIVE_LOW>;
-			linux,default-trigger = "heartbeat";
 		};
 
 		red {
-- 
2.7.4

